.vscode/settings.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "svg.preview.background": "custom"
```

Comment:
This is a setting for custom VSCode extension. So you should instead use your profile settings in VSCode instead of project setting.